pkg/yang/parse_test.go

```diff
@@ -105,7 +105,7 @@ foo "\\ \S \n";
 pattern "\\ \S \n";
 `,
 			out: []*Statement{
-				SA("pattern", `\\ \S 
+        SA("pattern", `\\ \S 
```

Comment:
Ah, apologies, I had issues with an editor removing other whitespace, and forgot to subsequently run `gofmt`. Fixed.